### PR TITLE
add env reader

### PR DESCRIPTION
**Pull request by RachidEddahaPrivate**
Context:
This set of changes introduces a file containing a function for reading environment variables using the Viper library:
https://github.com/spf13/viper

Previously:
- PR2: add git ignore and config env example (https://github.com/RachidEddahaPrivate/task-golang/pull/2)

In this PR:
A file has been added that includes a function for reading environment variables using the Viper library. We need to read the configuration variables to give the final user more flexibility when using this application.

Risk:
The function LoadConfiguration() is not called any where. No risk. The only problem could be if the file of the configuration is not in the cmd folder.

How to revert:
To revert these changes, remove the file configuration.go

Testing:
To validate these changes, execute the project in a local development environment. Call the function in the file main and confirm that the newly added function effectively reads environment variables using the Viper library.

### CODE REVIEW

File: pkg/configuration/configuration.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	LogSaveFile bool   `mapstructure:"LOG_SAVE_FILE"`
+}
+
+type ConfigEcho struct {
+	AllowedOrigins   string `mapstructure:"ALLOWED_ORIGINS" validate:"required"`
+	AllowCredentials bool   `mapstructure:"ALLOW_CREDENTIALS"`
+	AddressEcho      string `mapstructure:"ADDRESS_ECHO" validate:"required"`
+}
+
+func LoadConfiguration() (configuration Config, err error) {
```
Comment (ivaste) [inline pkg/configuration/configuration.go:31]: unit tests for this?

Comment (ivaste) [reply pkg/configuration/configuration.go:31]: Example of tabular unit tests 
https://github.com/golang/go/blob/master/src/fmt/errors_test.go

Comment (RachidEddahaPrivate) [reply pkg/configuration/configuration.go:31]: I added this very basic unit test for the function, as all the parameters are embedded in the file, and it's challenging to test with different parameters.

Comment (RachidEddahaPrivate) [reply pkg/configuration/configuration.go:31]: Added test unit

**Pull request merged**